<launch>

  <node name="ukf_localization_node" pkg="robot_localization" type="ukf_localization_node">
    <rosparam file="$(find nav_configs)/config/turtlebot_localization.yaml" command="load"/>
  </node>
  
  <node name="navsat_transform_node" pkg="robot_localization" type="navsat_transform_node">
    <param name="magnetic_declination_radians" value="0"/>
    <param name="yaw_offset" value="0"/>
   <!-- <param name="broadcast_utm_transform" value="true"/> -->
    <remap from="/imu/data" to="mobile_base/sensors/imu_data" />
    <remap from="/gps/fix" to="fix" />
    <remap from="/odometry/filtered" to="odometry/filtered" />
  </node>
  
</launch>
